lib/oneview-sdk/image-streamer/resource/api300/os_volumes.rb

```diff
@@ -17,6 +17,32 @@ module API300
       # OS Volumes resource implementation for Image Streamer
       class OsVolumes < Resource
```

Comment:
@jsmartt , I will correct the name class.

Regarding CHANGELOG, what was decided on the release? Will one soon be released with bugfix and enhancements or will they wait for the i3s features?